index.html

```diff
@@ -112,6 +112,17 @@
 )</pre>
     </li>
 
+    <li id="long-text">
```

Comment:
Rebase isn't quite right, this change is already in `master` and shouldn't appear in the PR.

Reply:
Currently, this is the branch in your fork. Since you're the collaborator, you should be able to create branches directly for `limonte/sweetalert2` repo. If you do so I will be able to help with rebasing.